app/views/posts/index.html.erb

```diff
@@ -9,9 +9,13 @@
 <% @list_posts.each do |post| %>
   <p><%= User.find(post.user_id).username %></p>
   <p><%= post.message %></p>
-  <%= link_to 'Like', post, method: 'patch' %>
+  <%= link_to image_tag('likebutton.png', width: 14, height: 14), post, method: 'patch' %>
```

Comment:
replaced link to text “like” to image stored in assets with image_tag right?

Reply:
Yes